compiler/compile/OMRCompilation.hpp

```diff
@@ -627,6 +627,8 @@ class OMR_EXTENSIBLE Compilation
    int32_t getInlinedCalls() { return _inlinedCalls; }
    void incInlinedCalls() { _inlinedCalls++; }
 
+   TR_ExternalRelocationTargetKind getReloTypeForMethodToBeInlined(TR_VirtualGuardSelection *guard, TR::Node *callNode, TR_OpaqueClassBlock *receiverClass) { return TR_NoRelocation; }
+
```

Comment:
Should there be an assert error if this ever gets returned in practice ? I understand that it is not expected to be called currently since a downstream project is expected to override it.